app/gosqs/gosqs.go

```diff
@@ -843,9 +843,22 @@ func SetQueueAttributes(w http.ResponseWriter, req *http.Request) {
 }
 
 func getMessageResult(m *app.Message) *app.ResultMessage {
-	attrs := []*app.ResultMessageAttribute{}
+	msgMttrs := []*app.ResultMessageAttribute{}
 	for _, attr := range m.MessageAttributes {
-		attrs = append(attrs, getMessageAttributeResult(&attr))
+		msgMttrs = append(msgMttrs, getMessageAttributeResult(&attr))
+	}
+
+	attrsMap := map[string]string{
```

Comment:
what is the attributes "SenderId" and  ApproximateReceiveCount do seems you are settings them to "fake" values and then using those as attributes??

Reply:
I do not see also in the docs where there is a distinction between "attributes' and "messageattributes"??